### PR TITLE
Aj resethead

### PR DESCRIPTION
**Pull request by anoopj08**

### CODE REVIEW

**Comment by anoopj08** [comment]
Screen Shot 2019-04-17 at 5 20 50 PM

File: lamby/controllers/models.py [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
-        object_link=fs.get_link(f'{project_id}/{commit_id}')
+        object_link=fs.get_link(f'{project_id}/{commit_id}'),
+        commits=Commit.query.filter_by(
+            project_id=project_id, filename=commit[0].filename),
+        project_id=project_id,
+        current_commit=commit_id,
+        reset_head_form=HeadForm()
     )
+
+@models_blueprint.route('resethead/<string:project_id>/<string:commit_id>')
```
Comment (ammuh) [inline lamby/controllers/models.py]: nit: change head might be better naming

**Comment by pkoushik** [comment]
Did you ever get this working? I know we had a few issues yesterday.

**Comment by pkoushik** [comment]
@anoopj08 I fixed the issue that we had on Wednesday, let me know what you think. I also removed the form.

File: lamby/controllers/models.py (hunk 1 above)
Comment (pkoushik) [reply lamby/controllers/models.py]: @anoopj08 can you make this change so ammu can approve it.

**Pull request merged**